apps/web/src/components/WalletConnect/ProjectInfoCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { Avatar, Card, Link } from "@chakra-ui/react";
+import { type SignClientTypes } from "@walletconnect/types";
+
+import { PencilIcon } from "../../assets/icons";
+
+/**
+ * Types
+ */
+interface IProps {
+  metadata: SignClientTypes.Metadata;
+  intention?: string;
+}
+
+/**
+ * Components
+ */
+export default function ProjectInfoCard({ metadata, intention }: IProps) {
+  const { icons, name, url } = metadata;
+
+  return (
+    <div style={{ textAlign: "center" }}>
+      <div>
```

Comment:
instead of <div> <span> and some other default tags we use chacra components (e.g., <box>) 

please check other components that we already have or ping in the chat for help